nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java

```diff
@@ -420,7 +433,11 @@ private String generateInsert(final JsonNode rootNode, final Map<String, String>
 
         final StringBuilder sqlBuilder = new StringBuilder();
         int fieldCount = 0;
-        sqlBuilder.append("INSERT INTO ").append(tableName).append(" (");
+        sqlBuilder.append("INSERT INTO ");
+        if (quoteTableName) sqlBuilder.append(schema.getQuotedIdentifierString());
```

Comment:
Quick remark: could you change to do it similarly to column names? Something like:
````java
            if(quoteTableName){
                sqlBuilder.append(schema.getQuotedIdentifierString())
                        .append(tableName)
                        .append(schema.getQuotedIdentifierString());
            } else {
                sqlBuilder.append(tableName);
            }
````
This way you avoid two if evaluations.

Also, since you are going to make a change, could you update this same piece of code for column names to avoid the negation check in the if statement? To have:
````java
            if(escapeColumnNames){
                sqlBuilder.append(schema.getQuotedIdentifierString())
                        .append(normalizedColName)
                        .append(schema.getQuotedIdentifierString());
            } else {
                sqlBuilder.append(normalizedColName);
            }
````
I believe this is more efficient.

Otherwise, LGTM.